geode-pulse/src/main/java/org/apache/geode/tools/pulse/internal/PulseAppListener.java

```diff
@@ -44,18 +44,37 @@
 // @WebListener
 public class PulseAppListener implements ServletContextListener {
   private static final Logger logger = LogManager.getLogger();
-  private final ResourceBundle resourceBundle = Repository.get().getResourceBundle();
   private static final String GEODE_SSLCONFIG_SERVLET_CONTEXT_PARAM = "org.apache.geode.sslConfig";
 
-  private Properties pulseProperties;
-  private Properties pulseSecurityProperties;
+  private final boolean sysIsEmbedded;
+  private final Repository repository;
+  private final Function<String, Properties> functionLoadProperties;
```

Comment:
Consider `propertiesFileLoader`. The word 'function' adds little meaning.

Reply:
Thanks! Done.